database/seeders/AccountSeeder.php

```
@@ -36,5 +36,13 @@ public function run(): void
         }
         $ids = range(311, 388);
         Account::whereIn('id', $ids)->delete();
+        
+        $taxableAccountsNumbers = [
```

PHPMD (2.15.0)

Avoid excessively long variable names like $taxableAccountsNumbers. Keep variable name length under 20. (Naming Rules)

(LongVariable)